backends/beats/winlogbeat/render.go

```diff
@@ -152,3 +161,64 @@ func (wlbc *WinLogBeatConfig) ValidateConfigurationFile() bool {
 
 	return true
 }
+
+func (wlbc *WinLogBeatConfig) runMigrations(bc *beats.BeatsConfig) {
```

Comment:
It looks like we are duplicating the code from the file beat. Any way to move this into a shared function?